Code/Framework/AzToolsFramework/AzToolsFramework/FocusMode/FocusModeInterface.h

```diff
@@ -17,6 +17,8 @@
 
 namespace AzToolsFramework
 {
+    using EntityIdList = AZStd::vector<AZ::EntityId>;
```

Comment:
I was going to make this exact same comment, but then did a quick search and it actually appears all over 😅 

```
Gems\Prefab\PrefabBuilder\PrefabGroup\PrefabGroupBehavior.cpp
247:        using EntityIdList = AZStd::vector<AZ::EntityId>;

Gems\LyShine\Code\Editor\EntityHelpers.h
14:    using EntityIdList = AzToolsFramework::EntityIdList;

Code\Framework\AzToolsFramework\AzToolsFramework\Prefab\PrefabPublicRequestBus.h
23:    using EntityIdList = AZStd::vector<AZ::EntityId>;

Code\Framework\AzToolsFramework\AzToolsFramework\Prefab\PrefabPublicInterface.h
18:    using EntityIdList = AZStd::vector<AZ::EntityId>;

Code\Framework\AzToolsFramework\AzToolsFramework\Entity\SliceEditorEntityOwnershipServiceBus.h
23:    using EntityIdList = AZStd::vector<AZ::EntityId>;

Code\Framework\AzToolsFramework\AzToolsFramework\Entity\EditorEntityTransformBus.h
15:    using EntityIdList = AZStd::vector<AZ::EntityId>;

Code\Framework\AzToolsFramework\AzToolsFramework\API\EditorEntityAPI.h
17:    using EntityIdList = AZStd::vector<AZ::EntityId>;

Code\Framework\AzToolsFramework\AzToolsFramework\API\ToolsApplicationAPI.h
57:    using EntityIdList = AZStd::vector<AZ::EntityId>;

Code\Framework\AzToolsFramework\AzToolsFramework\API\EntityCompositionNotificationBus.h
18:    using EntityIdList = AZStd::vector<AZ::EntityId>;

Code\Framework\AzFramework\AzFramework\Entity\EntityOwnershipService.h
28:    using EntityIdList = AZStd::vector<AZ::EntityId>;
```

Not suggesting adding another is the best idea in the world but maybe a separate PR to clean-up some of the duplicates might be something we can do separately 🙂

Reply:
(performed using the amazing [ripgrep](https://github.com/BurntSushi/ripgrep) 😁 - `rg "using EntityIdList" -t cpp`

Reply:
Agreed, it's been tricky since we don't want to include unused interfaces just for this definition.

I see Prefabs have Helper files with just definitions that are included in the interfaces, so maybe we could do something similar.

Reply:
I have managed to move all of these definitions to a single header as part of this change: https://github.com/o3de/o3de/pull/7630

Reply:
That's brilliant! 😄 Thanks @AMZN-daimini Awesome work! 😎